docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -0,0 +1,92 @@
+# Feature Band Support: Work Plan
+
+This document builds on top of [Managing SDK Bands](https://github.com/dotnet/dotnet/blob/main/docs/VMR-Managing-SDK-Bands.md) with a concrete work plan for providing feature band support in the VMR. Please read that doc as a prerequisite.
+
+Document convention:
+* There can be a variable number of feature band releases (e.g. 10.0.1xx, 10.0.2xx, 10.0.3xx, etc). For purposes of this document, we mainly care about the distinction between the 1xx branch and all the other non-1xx feature band branches. For simplicity, 2xx will be used as a notation to mean any non-1xx feature band branch.
+
+## Plan Details
+
+### Branch Management
+
+The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation.
+
+There should be tooling to help with this exclusion process for the VMR maintainers. Similarly, the associated projects in the `repo-projects` directory should be deleted along with updating the appropriate `RepositoryReference` usages.
+
+### Dependency Flow
+
+Repos in the 2xx branch of the VMR which have dependencies on repos solely defined in the 1xx branch (e.g. `sdk` repo's dependency on `runtime`) will need to have dependency flow that reflects the version of the package produced from the VMR's 1xx branch.
```

Comment:
Open question is how/where the 1xx dependencies are defined in the 2xx branch

Reply:
Can we have something like `/Dependencies/Dependency/Branch` in `Version.Details.xml`? If present it would point at dependent VMR branch, and will be updated when we flow lower SDK level to a higher one.

Reply:
Product dependencies in the root of the 2xx eng/V.D.xml?